chaco/tests/test_speedups.py

```diff
@@ -55,6 +66,5 @@ def test_selection_range(self):
 
     def _get_func(self):
         return self.module.scatterplot_gather_points
-    func = property(_get_func)
-
 
+    func = property(_get_func)
```

Comment:
we can get rid of this old way of setting up properties - use `@property` instead of `property(...)`

Reply:
#609